h/templates/deform/mapping_item.jinja2

```diff
@@ -46,7 +46,9 @@
 
 {%- if field.error and not field.widget.hidden -%}
   <ul class="form-input__error-list">
-  {% for msg in field.error.messages() -%}
+  <!-- Only show the first error message for each field. -->
+  <!-- https://github.com/hypothesis/product-backlog/issues/581 -->
```

Comment:
We should use Jinja2 comments (`{# ... #}`) rather than HTML comments since Jinja comments don't show up in the rendered output.